docs/api/build_proto.md

```diff
@@ -72,14 +72,14 @@ There are many tools for building proto in Rust, we use [tonic](https://github.c
 1.  Create project
 
     ```bash
-    $ cargo new --lib vald-grpc
+    cargo new --lib vald-grpc
     ```
 
 1.  Edit `Cargo.toml`
 
     ```bash
-    $ cd vald-grpc
-    $ vim Cargo.toml
+    cd vald-grpc && \
+    vim Cargo.toml
     ---
```

Comment:
I think we should separate this in another block

Reply:
All this kind of changes is for the clipboard function in the public document.